### PR TITLE
Dual-stack docs correction

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Requested by @aojea who noticed this slight error in the docs & @khenidak who confirmed this correction.

Signed-off-by: Bridget Kromhout <bridget@kromhout.org>

### CODE REVIEW

File: content/en/docs/concepts/services-networking/dual-stack.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
 
 If your cluster has dual-stack enabled, you can create {{< glossary_tooltip text="Services" term_id="service" >}} which can use IPv4, IPv6, or both. 
 
-The address family of a Service defaults to the address family of the first service cluster IP range (configured via the `--service-cluster-ip-range` flag to the kube-controller-manager).
+The address family of a Service defaults to the address family of the first service cluster IP range (configured via the `--service-cluster-ip-range` flag to the kube-api-server).
```
Comment (aramase) [inline content/en/docs/concepts/services-networking/dual-stack.md:77]:
```suggestion
The address family of a Service defaults to the address family of the first service cluster IP range (configured via the `--service-cluster-ip-range` flag to the kube-apiserver).
```

Comment (bridgetkromhout) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Thanks!

**Review by aramase — COMMENTED** [review]
/lgtm

**Comment by aramase** [comment]
/hold

for other reviews

**Pull request merged**